### ISSUE TITLE
tepid-discover does not work with genome assemblies with only seqs named 'scaffold'

### ISSUE THREAD

**Issue by Shellfishgene** [ListerLab/TEPID#4]
Hi!
The [this code](https://github.com/ListerLab/TEPID/blob/master/tepid/tepid.py#L715-L718) in the calc_cov method skips over any sequences that have scaffold in the name. I assume this is to only work with "chr" sequences in Arabidopsis or something? However this breaks tepid-discover for any other asseblies that only have scaffolds that are named as such. Just removing the if statement seems to work for me.

**Comment by timoast** [comment]
Yes, in analyses I worked on I found including many of the scaffolds for certain genomes was introducing too many false positives and so I removed them. I agree it would be much better to have this as a user option, I don't have much time to work on this right now but would be open to a PR for this
